src/index.html

```diff
@@ -7,6 +7,298 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="value" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="/apple" class="nav__link is-active">apple</a>
```

Comment:
```suggestion
            <a 
              href="/apple" 
              class="nav__link 
              is-active"
            >
              apple
            </a>
```
check your code style